### ISSUE TITLE
blockdev autotest intermittent failure - timeout

### ISSUE THREAD

**Issue by jimharris** [spdk/spdk#910]
I've seen this several times.  The blockdev test gets aborted at 10 minutes since it hasn't completed.  It's odd that we see a 2 minute delay during the test for seemingly no real reason:

0000:00:04.4 (8086 3c24): vfio-pci -> ioatdma
0000:80:04.4 (8086 3c24): vfio-pci -> ioatdma
0000:00:04.5 (8086 3c25): vfio-pci -> ioatdma
0000:80:04.5 (8086 3c25): vfio-pci -> ioatdma
0000:00:04.6 (8086 3c26): vfio-pci -> ioatdma
0000:80:04.6 (8086 3c26): vfio-pci -> ioatdma
0000:00:04.7 (8086 3c27): vfio-pci -> ioatdma
0000:80:04.7 (8086 3c27): vfio-pci -> ioatdma
 00:07:21	# /var/jenkins/workspace/Other/blockdev_autotest/spdk/scripts/setup.sh cleanup
ls: cannot access '/dev/shm/*': No such file or directory
Cleaning
Removing:    /run/dpdk/spdk0/config
Removing:    /run/dpdk/spdk0/fbarray_memseg-2048k-0-0
Removing:    /run/dpdk/spdk0/fbarray_memseg-2048k-0-1
Removing:    /run/dpdk/spdk0/fbarray_memseg-2048k-0-2
Removing:    /run/dpdk/spdk0/fbarray_memseg-2048k-0-3
Removing:    /run/dpdk/spdk0/fbarray_memseg-2048k-1-0
Removing:    /run/dpdk/spdk0/fbarray_memseg-2048k-1-1
Removing:    /run/dpdk/spdk0/fbarray_memseg-2048k-1-2
Removing:    /run/dpdk/spdk0/fbarray_memseg-2048k-1-3
Removing:    /run/dpdk/spdk0/fbarray_memzone
Removing:    /run/dpdk/spdk0/hugepage_info
Removing:    /run/dpdk/spdk0/mp_socket
Removing:    /var/run/dpdk/spdk0
Clean
  00:09:16	# uname -s
 00:09:16	# '[' Linux = Linux ']'
 00:09:16	# grep -q '#define SPDK_CONFIG_IGB_UIO_DRIVER 1' /var/jenkins/workspace/Other/blockdev_autotest/spdk/include/spdk/config.h
 00:09:16	# modprobe -r uio_pci_generic
 00:09:16	# rm -rf /var/tmp/asan_suppression_file
 00:09:16	# timing_exit cleanup
 00:09:16	# xtrace_disable
 00:09:16	# PREV_BASH_OPTS=ehxBE
 00:09:16	# set +x

Basically the cleanup took 2 minutes???  At first I thought this was because a VM got swapped out, but I think it's a bare metal system:

Building remotely on GP4 (SingleDisk PhyBlockdev) in workspace /var/jenkins/workspace/Other/blockdev_autotest

**Comment by jimharris** [comment]
Another instance of this.  On same system as last one (GP4):

https://dqtibwqq6s6ux.cloudfront.net/results/autotest-per-patch/builds/39983/archive/blockdev_autotest/build.log

**Comment by darsto** [comment]
Happens on other jobs as well - not only blockdev. Still no root cause.

**Comment by karlatec** [comment]
FYI - blockdev_autotest was renamed a couple of days ago to crypto_autotest.

I was investigating https://github.com/spdk/spdk/issues/939 a couple of days ago and found out that there were some issues with NFS network which are now fixed. This might have been somehow related to #910 as I haven't seen a timeout since last tuesday / wednesday.
Another thing is that GP4 had a faulty disk which was replaced about the same time as the NFS fix was done.

**Comment by karlatec** [comment]
Check the job build logs again. Could only find two timeouts, and both builds were run of "autotest nightly failing", so I'm pretty sure now that this issue is fixed now.

**Comment by karlatec** [comment]
Checked again, no timeouts on crypto_autotest job last week. Adding to bugscrub, we will close it then.

**Comment by jimharris** [comment]
Bug scrub 09/24: @karlatec fixed an NFS issue that we think would have addressed this issue.  Issue not seen since then, so closing this issue.  We can reopen it again if it appears again.
